src/features/auth/NetworkSelector.tsx

```diff
@@ -152,7 +153,7 @@ export const NetworkSelector = () => {
             background: '$dim',
             mt: '$sm',
             p: '$sm',
-            width: 'var(--radix-popover-trigger-width)',
```

Comment:
@wingfeatherwave  the --radix-popover-trigger-width make it resizes with different screens